src/components/common/CertifySection.vue

```diff
@@ -61,7 +61,7 @@ export default class CertifySection extends Mixins(DateMixin, SharedMixin) {
   @Prop({ default: false }) readonly disableEdit: boolean
 
   /** Called when component is mounted. */
-  mounted (): void {
+  protected mounted (): void {
```

Comment:
`mounted` should not be called externally, but we might want a child class to override it.